cmd/kube-apiserver/app/options/features.go

```diff
@@ -0,0 +1,18 @@
+package options
+
+import (
+	genericfeatures "k8s.io/apiserver/pkg/features"
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	"k8s.io/kubernetes/pkg/features"
+)
+
+var featureGates = map[utilfeature.Feature]utilfeature.FeatureSpec{
+	features.ExternalTrafficLocalOnly: {Default: true, PreRelease: utilfeature.GA},
```

Comment:
feature gates used by multiple components are common... this approach makes it really easy to have conflicting default values in different components at a single build level... how are you detecting/preventing that?